libdivide.h

```diff
@@ -13,7 +13,14 @@
 // unsigned type, result still unsigned
 #pragma warning(disable: 4146)
 #define LIBDIVIDE_VC
+#if _M_IX86_FP == 2
+#define __SSE2__ 1
```

Comment:
Let's not #define __SSE2__, that's the purview of the compiler. Better to define LIBDIVIDE_USE_SSE2 directly here.